src/test/java/transmitter/CheckTextTest.java

```diff
@@ -0,0 +1,148 @@
+/**
+ *
+ */
+package transmitter;
+
+import static org.junit.Assert.*;
+
+import java.lang.reflect.Method;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author z00s600078
+ *
+ */
+public class CheckTextTest {
+
+	private CheckText ct;
+
+	@Before
+	public void setUp() throws Exception {
+		ct = new CheckText();
+	}
+
+    @After
+    public void tearDown() throws Exception {
+    	ct = null;
+    }
+
+
+	/**
+	 * {@link transmitter.CheckText#CheckText()} のためのテスト・メソッド。
+	 */
+	@Test
+	public void testCheckText(){
+	}
+
+
+	/**
+	 * {@link transmitter.CheckText#validate(java.lang.String)} のためのテスト・メソッド。
+	 */
+
+	/* 正常系 半角英数かつ16文字以下かつ空白を含まない場合trueを返す */
+	@Test
+	public void testValidate1() {
+		assertEquals(true, ct.validate("hello"));
```

Comment:
The assertEquals is old method, we use assertThat and Matcher.
http://javazuki.com/articles/assert-that-matcher-introduction.html